src/test/java/wbooks/userdata/userdata.feature

```diff
@@ -0,0 +1,41 @@
+Feature: get data from specific user
+
+  Background:
+    * url 'https://wbooks-api-stage.herokuapp.com/api/v1'
+
+    * def login = call read('classpath:wbooks/login/login.feature@login')
+    * def authToken = login.access_token
+
+    * def userDataSchema =
```

Comment:
be more specifig in the data type of each field

Reply:
Added types to schema now. Not sure if the schema is the correct one tough, its done from the API current answer